docroot/profiles/custom/cgov_site/modules/custom/cgov_yaml_content/content/svpc_what-is-liver-cancer.content.yml

```diff
@@ -76,7 +76,7 @@
   field_pdq_audience__ES:
     value: Patients
   field_pdq_summary_type__ES:
-    value: Overview
+    value: Genetics
```

Comment:
Why did this change?

Reply:
Bad YAML - Overview is not a valid option under summary type in drupal - updated yaml to match production